qiskit/synthesis/clifford/clifford_decompose_greedy.py

```diff
@@ -32,22 +32,28 @@
 
 
 def synth_clifford_greedy(clifford):
-    """Decompose a Clifford operator into a QuantumCircuit based on the
-    greedy Clifford compiler that is described in Appendix A of
-    Bravyi, Hu, Maslov and Shaydulin.
+    """Decompose a :class:`.Clifford` operator into a :class:`.QuantumCircuit` based
+    on the greedy Clifford compiler that is described in Appendix A of
+    Bravyi, Hu, Maslov and Shaydulin [1].
 
     This method typically yields better CX cost compared to the Aaronson-Gottesman method.
 
+    .. note::
```

Comment:
Maybe instead of a formal note, it can just be a sentence like 
"Note that ..."